terraform/redis-enterprise.tf

```diff
@@ -0,0 +1,105 @@
+#
+# Provision for GCP MP subscription
+#
+# Ensure the following variables in your shell environment:
+# export GOOGLE_APPLICATION_CREDENTIALS=<your Google APIs credentials>
+# export REDISCLOUD_ACCESS_KEY=<your Redis Enterrpise Cloud api access key>
+# export REDISCLOUD_SECRET_KEY=<your Redis Enterprise Cloud secret key>
```

Comment:
**Question:**
Sorry, Redis Enterprise is new to me, so I'm confused. :)
If the Terraform takes care of _creating_ the Redis subscription (`rediscloud_subscription`), how do we get the `REDISCLOUD_ACCESS_KEY` and/or `REDISCLOUD_SECRET_KEY` associated with the Redis subscription?
Is there a preliminary step to running the Terraform? Is that step everything inside https://github.com/Redislabs-Solution-Architects/redis-enterprise-cloud-gcp/blob/main/marketplace/gcp/redis-enterprise.md?